### PR TITLE
solution created

### PR DESCRIPTION
**Pull request by dreeeamcatcher**

### CODE REVIEW

File: src/main/java/mate/controller/AddDriverToCarController.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private CarService carService = (CarService) injector.getInstance(CarService.class);
+    private DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long carId = Long.valueOf(req.getParameter("carId"));
+        Long driverId = Long.valueOf(req.getParameter("driverId"));
```
Comment (Rommelua) [inline src/main/java/mate/controller/AddDriverToCarController.java:25]: Use the same style 
image

```suggestion
        Long carId = Long.valueOf(req.getParameter("car_id"));
        Long driverId = Long.valueOf(req.getParameter("driver_id"));
```

File: src/main/java/mate/controller/AddDriverToCarController.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import mate.service.DriverService;
+
+@WebServlet("/cars/drivers/add")
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private CarService carService = (CarService) injector.getInstance(CarService.class);
+    private DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
```
Comment (Rommelua) [inline src/main/java/mate/controller/AddDriverToCarController.java:22]: This is not GET method
```suggestion
    public void doPost(HttpServletRequest req, HttpServletResponse resp)
```

File: src/main/java/mate/controller/AddDriverToCarController.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
+package mate.controller;
```
Comment (Rommelua) [inline src/main/java/mate/controller/AddDriverToCarController.java:1]: add packages:
car
driver
manufacturer

File: src/main/java/mate/controller/SelectDriverToAddToCarController.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet("/cars/drivers/select_to_add")
+public class SelectDriverToAddToCarController extends HttpServlet {
```
Comment (Rommelua) [inline src/main/java/mate/controller/SelectDriverToAddToCarController.java:17]: delete this controller and add GET method to AddDriverToCarController

File: src/main/java/mate/controller/DeleteDriverFromCarController.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+import mate.service.DriverService;
+
+@WebServlet("/cars/drivers/delete")
+public class DeleteDriverFromCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private CarService carService = (CarService) injector.getInstance(CarService.class);
+    private DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
```
Comment (Rommelua) [inline src/main/java/mate/controller/DeleteDriverFromCarController.java:22]: post

File: src/main/java/mate/controller/SelectDriverToDeleteFromCarController.java [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+
+@WebServlet("/cars/drivers/select_to_delete")
+public class SelectDriverToDeleteFromCarController extends HttpServlet {
```
Comment (Rommelua) [inline src/main/java/mate/controller/SelectDriverToDeleteFromCarController.java:16]: delete this controller and add GET method to DeleteDriverFromCarController

**Review by Rommelua — CHANGES_REQUESTED** [review]

**Comment by dreeeamcatcher** [comment]
Didn't get the following idea:
          "add packages:
          car
          driver
          manufacturer"
did you mean to import the package, not the particular classes?

**Review by aaananas — APPROVED** [review]

**Pull request closed without merging**